nullaway/src/main/java/com/uber/nullaway/fixserialization/location/MethodParameterLocation.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.uber.nullaway.fixserialization.location;
+
+import com.google.common.base.Preconditions;
+import com.sun.tools.javac.code.Symbol;
+import javax.lang.model.element.ElementKind;
+
+/** subtype of {@link AbstractFixLocation} targeting a method parameter. */
+public class MethodParameterLocation extends AbstractFixLocation {
+
+  /** Symbol of the targeted method. */
+  protected final Symbol.MethodSymbol enclosingMethod;
```

Comment:
Why protected? Did I miss a subclass of this?

Reply:
7271a3e6dff551ac7861bb47a24506fef5c24efd